### PR TITLE
Create exampleSchema

### PR DESCRIPTION
**Pull request by Flanker-shyam**
State
college
course
QuestionPapers

### CODE REVIEW

File: exampleSchema [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
+const stateSchema = new Schema({
+  name: {
+    type: String,
+    required: true,
+    unique: true
+  }
+});
```
Comment (debasishbsws) [inline exampleSchema:8]: Why, Do we need a state schema? we are not classifying institutes via the state.

**Review by debasishbsws — COMMENTED** [review]
It would be better instead of code we have the schema in JSON format

File: exampleSchema (hunk 1 above)
Comment (Flanker-shyam) [reply exampleSchema:8]: In that case it is not required, I planned it so to upscale this project later.

Like the case when we will be having many institutions and the user will be able to search with the help of state.

Comment (debasishbsws) [reply exampleSchema:8]: Okay we could have that no problem

**Comment by Flanker-shyam** [comment]
> It would be better instead of code we have the schema in JSON format

Okay, will look into this modification.

**Comment by debasishbsws** [comment]
No need. We will be using sql

**Pull request closed without merging**